examples/basics_cpp/pebble.cpp

```diff
@@ -0,0 +1,54 @@
+#include "pebble.h"
```

Comment:
Why is this file copied? Can you update the wscript to include this from the src/ directory instead?